src/ui/server/controllers/insurance/policy/broker-addresses/index.ts

```diff
@@ -0,0 +1,31 @@
+import { PAGES } from '../../../../content-strings';
+import { TEMPLATES } from '../../../../constants';
+import { INSURANCE_ROUTES } from '../../../../constants/routes/insurance';
+import insuranceCorePageVariables from '../../../../helpers/page-variables/core/insurance';
+import getUserNameFromSession from '../../../../helpers/get-user-name-from-session';
+import { Request, Response } from '../../../../../types';
+
+const { PROBLEM_WITH_SERVICE } = INSURANCE_ROUTES;
+
+export const PAGE_CONTENT_STRINGS = PAGES.INSURANCE.POLICY.BROKER_ADDRESSES;
+
+export const TEMPLATE = TEMPLATES.INSURANCE.POLICY.BROKER_ADDRESSES;
+
+/**
+ * Render the Broker addresses page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.render} Broker addresses page
+ */
+export const get = (req: Request, res: Response) => {
+  const { application } = res.locals;
+
+  if (!application) {
+    return res.redirect(PROBLEM_WITH_SERVICE);
```

Comment:
This is in place for every single controller - would like to add a `console.error` in it's own ticket/PR. In fact. might be able to do this in one place in middleware instead of individual controllers.

OK to look into this separately? Will create a ticket.

Reply:
I have created a ticket for this, EMS-4013